packages/phrases/src/locales/en/errors.ts

```diff
@@ -111,6 +111,8 @@ const errors = {
     can_not_modify_target: 'The connector target can not be modified.',
     multiple_target_with_same_platform:
       'You can not have multiple social connectors that have same target and platform.',
+    cannot_change_metadata_for_non_standard_connector:
+      "Should not config 'metadata' for a non standard connector.",
```

Comment:
IMO, we don't necessarily need to expose the `standard connector` concept to any end users. What about just `Connector metadata cannot be changed.`